common/config/rush/pnpm-lock.yaml

```diff
@@ -8698,7 +8699,7 @@ packages:
   file:projects/communication-phone-numbers.tgz:
     dependencies:
       '@azure/core-tracing': 1.0.0-preview.11
-      '@azure/identity': 1.3.0
+      '@azure/identity': 2.0.0-beta.3
```

Comment:
Not sure how this is going to work cause I updated the lock file in my PR as well. Might have a conflict but I will deal with it when this gets merged.

Reply:
I think it'll be based on whoever merges in first...hmmm